strawberry/relay/types.py

```diff
@@ -705,6 +699,8 @@ def resolve_node(cls, node: Any, *, info: Info, **kwargs: Any) -> NodeType:
                 connection's `NodeType`
             info:
                 The strawberry execution info resolve the type name from
+            **kwargs:
+                Additional arguments passed to the resolver
```

Comment:
nit: as above again again

```suggestion
                connection's `NodeType`.
            info:
                The strawberry execution info resolve the type name from.
            **kwargs:
                Additional arguments passed to the resolver.
```